packages/oidc-middleware/README.md

```diff
@@ -154,6 +154,8 @@ app.get('/', (req, res) => {
 });
 ```
 
+Note: The default scope of `openid` will only return the `sub` claim.  To obtain more information about the user, set the `scope` option to `openid profile` when creating the middleware.
```

Comment:
consider based on https://developer.okta.com/standards/OIDC/index.html#scope-dependent-claims-not-always-returned